src/main/java/com/microsoft/lst_bench/common/TaskExecutor.java

```diff
@@ -58,10 +58,15 @@ public void executeTask(Connection connection, TaskExec task, Map<String, Object
             connection.execute(StringUtils.replaceParameters(statement, values).getStatement());
           } catch (Exception e) {
             LOGGER.error("Exception executing statement: " + statement.getId());
-            writeStatementEvent(statementStartTime, statement.getId(), Status.FAILURE);
+            writeStatementEvent(
+                statementStartTime,
+                statement.getId(),
+                Status.FAILURE,
+                e.getMessage() + "; " + e.getStackTrace());
```

Comment:
Have you checked that the stack trace coming from `getStackTrace` is unrolled and printed properly? Iirc there is some additional handling that needs to be done for that, but I could be wrong.